### PR TITLE
allow for editing application-specific scopes

### PR DESCRIPTION
**Pull request by redterror**
Tiny PR to add the `scopes` field to the application's show/edit forms.  Doesn't fundamentally change anything but removes the requirement for editing things directly in the DB.

### CODE REVIEW

File: app/views/doorkeeper/applications/_form.html.erb [hunk 1]
```diff
@@ -27,6 +27,17 @@
...
   <% end %>
 
+  <%= content_tag :div, class: "form-group#{' has-error' if application.errors[:scopes].present?}" do %>
+    <%= f.label :scopes, class: 'col-sm-2 control-label' %>
+    <div class="col-sm-10">
+      <%= f.text_field :scopes, class: 'form-control' %>
+      <%= doorkeeper_errors_for application, :scopes %>
+      <span class="help-block">
+         <%= t('doorkeeper.applications.help.scopes') %>
+        </span>
```
Comment (tute) [inline app/views/doorkeeper/applications/_form.html.erb]: Funky indentation in these two lines.

File: app/views/doorkeeper/applications/show.html.erb [hunk 2]
```diff
@@ -12,6 +12,10 @@
 
     <p><code id="secret"><%= @application.secret %></code></p>
 
+    <h4><%= t('.scopes') %>:</h4>
+
```
Comment (tute) [inline app/views/doorkeeper/applications/show.html.erb]: We can delete this new line.

**Comment by tute** [comment]
This is great, left minor comments, will merge when addressed! Can you update the `NEWS.md` file please? Thank you!

**Comment by redterror** [comment]
Better?  :)

**Comment by tute** [comment]
Yeah! Will merge on green, thanks for this.

**Comment by redterror** [comment]
Derp, didn't do the NEWS.md.  Doing that now.

**Comment by tute** [comment]
Very well. Let's make it deluxe and squash the three commits together?

**Comment by redterror** [comment]
Drive around to the first window please.

**Comment by tute** [comment]
Beautiful. :tada:

**Pull request merged**